src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -1296,8 +1296,7 @@ private static DateTime ParseTimeOfDay(ReadOnlySpan<char> time)
             {
                 if (date[0] != 'J')
                 {
-                    // should be n Julian day format which we don't support.
-                    //
+                    // should be n Julian day format.
```

Comment:
The comment below says "February 29 is counted in leap years". but the man page for tzset says:
```
       Jn     This specifies the Julian day with n between 1 and 365.
              Leap days are not counted.  In this format, February 29
              can't be represented; February 28 is day 59, and March 1
              is always day 60.
```
Am I looking at the wrong thing?

If leap years can't be counted, that would explain why it's not very useful past Feb 28...

Reply:
I think you got confused, this is handling the `n` format not the `Jn`. The `n` format goes from 0-365 and counts leap years.

See my comment below, but the Jn format is handled after this if.

Reply:
Oh..nuts☺️